src/benchmark/benchmark.cpp

```diff
@@ -1,6 +1,6 @@
 // Luanti
 // SPDX-License-Identifier: LGPL-2.1-or-later
-// Copyright (C) 2022 Minetest Authors
+// Copyright (C) 2022 Luanti Authors
```

Comment:
I believe we chose to intentionally not change this as the project was called "Minetest" when this file was created.